### PR TITLE
use new WebGLRenderer.outputEncoding

### PR DESCRIPTION
**Pull request by arpu**
https://github.com/mrdoob/three.js/wiki/Migration-Guide#r111--r112

### CODE REVIEW

**Comment by arpu** [comment]
@donmccurdy 
```
renderer.gammaOutput = true;
renderer.gammaFactor = 2.2;
```

should be removed?

**Comment by donmccurdy** [comment]
Yes — they should be replaced by `renderer.outputEncoding = THREE.sRGBEncoding`

**Comment by dmarcos** [comment]
Text failing

**Comment by arpu** [comment]
@dmarcos  any idea what is failing on the test? 

```
 ✖ change renderer colorManagement
      Firefox 73.0.0 (Linux 0.0.0)
    uncaught exception: AssertionError: expected undefined to be truthy (/tmp/src/core/a-node.js:9:0 <- /tmp/6b9e383798e24fc953d6862cb3025745.browserify:66793)
    require<[104]</module.exports<.prototype<.emit.value</<@/tmp/src/core/a-node.js:9:0 <- /tmp/6b9e383798e24fc953d6862cb3025745.browserify:66793:11365
    emitLoaded@/tmp/src/core/a-node.js:9:0 <- /tmp/6b9e383798e24fc953d6862cb3025745.browserify:66793:5800

```

File: tests/systems/renderer.test.js [hunk 1]
```diff
@@ -32,8 +31,8 @@ suite('renderer', function () {
     var sceneEl = createScene();
     sceneEl.setAttribute('renderer', 'colorManagement: true;');
     sceneEl.addEventListener('loaded', function () {
-      assert.ok(sceneEl.renderer.gammaOutput);
-      assert.equal(sceneEl.renderer.gammaFactor, 2.2);
+      assert.ok(sceneEl.renderer.outputEncodingt);
```
Comment (donmccurdy) [inline tests/systems/renderer.test.js]: typo on "Encodingt" here 🙂

**Comment by dmarcos** [comment]
@arpu Awesome. Thanks for taking care of the tests. @donmccurdy thanks for the feedback.

**Pull request merged**